backend/routes/api.php

```diff
@@ -17,7 +20,13 @@
 |
 */
 
-Route::resource('/post', UserController::class);
+Route::resource('/users', UserController::class);
+
+Route::resource('/categories', CategoryController::class);
+
+Route::resource('/lessons', LessonController::class);
+
+Route::resource('/words', WordController::class);
```

Comment:
This is good, you are using the Laravel `Route::resource()`, yet we need to use only the 5 because we are developing for API.
Ref: https://stackoverflow.com/questions/54721576/laravel-route-apiresource-difference-between-apiresource-and-resource-in-route
https://laravel.com/docs/9.x/eloquent-resources
Topic: Route resource vs Route API resource
```suggestion
Route::apiResource('/users', UserController::class);
Route::apiResource('/categories', CategoryController::class);
Route::apiResource('/lessons', LessonController::class);
Route::apiResource('/words', WordController::class);
```